tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -155,7 +155,7 @@ def test_ids_from_035__a_9_with_cern():
     assert expected == result['ids']
 
 
-def test_ids_from_035__a_9_with_malformed_cern():
+def test_ids_from_035__a_9_with_cern_cern():
```

Comment:
was this change intended? (now it does not make sense to me, from an out of context perspective, maybe I'm missing something)